script.js

```diff
@@ -0,0 +1,287 @@
+const UA =
+  "https://upload.wikimedia.org/wikipedia/commons/d/d2/Flag_of_Ukraine.png";
+
+const PL =
+  "https://upload.wikimedia.org/wikipedia/commons/1/12/Flag_of_Poland.svg";
+
+const NL =
+  "https://upload.wikimedia.org/wikipedia/commons/2/20/Flag_of_the_Netherlands.svg";
+
+const DE =
+  "https://upload.wikimedia.org/wikipedia/commons/b/ba/Flag_of_Germany.svg";
+
+const SP =
+  "https://upload.wikimedia.org/wikipedia/commons/9/9a/Flag_of_Spain.svg";
+
+const IT =
+  "https://upload.wikimedia.org/wikipedia/commons/0/03/Flag_of_Italy.svg";
+
+const USA =
+  "https://upload.wikimedia.org/wikipedia/commons/0/05/Flag_of_the_U.S..svg";
+
+const listProducts = [
+  {
+    productName: "Bread",
+    productData: {
+      weight: 200,
+      certificate: true,
+      dateOfExpiry: "30/10/2022",
+      sugarFree: true,
+    },
+    productProvider: "FOP Skyba R.R.",
+    productCountry: UA,
+    productPrice: 11.75,
+  },
+  {
+    productName: "Sugar",
+    productData: {
+      weight: 1000,
+      certificate: true,
+      dateOfExpiry: "21/12/2022",
+      sugarFree: true,
+    },
+    productProvider: "FOP Skyba R.R.",
+    productCountry: UA,
```

Comment:
```suggestion
    productCountry: getFlag('UA')
```

and so on. It will save you a lot of space and way more less code